surveys/2024-annual-survey/questions.md

```diff
@@ -414,11 +382,33 @@ Type: select all that apply (optional)
 - Achieving structured concurrency with async code
 - Writing executor-agnostic async code
 - Borrow checker not allowing valid code
+
+- (From here the additional one)
+- Runtime performance
+- Compile times
+- Binary size (of the final compiled artifact)
+- Memory usage (example: how much RAM rustc uses when compiling)
+- Disk space usage (example: the size of target folder)
+- Bugs in the compiler (that is, ICEs a.k.a. internal compiler errors, miscompilations, etc.)
+- Opaque / unclear compiler error messages
+- IDE experience
+- Debugging experience (example: unclear / misleading diagnostics)
+- Documentation (rustdoc, docs.rs)
+- Build system (cargo)
+- Package management (crates.io)
+- New language features
+- Rust language and standard library documentation
 - Other (please specify)
```

Comment:
Matrix questions in SH sadly cannot have the "Other" open category :( So we will need additional question anyway, to ask for open answers. This would mean that we wouldn't reduce question count by one, which is a bit annoying, but nothing too bad I guess.